Fakebook.Posts/Fakebook.Posts.RestApi/DTOs/FollowerDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Fakebook.Posts.RestApi.DTOs
+{
+    public class FollowDTO
+    {
+        [Required]
+        [DataType(DataType.EmailAddress)]
```

Comment:
this attribute doesn't add anything, [EmailAddress] already covers it.

Reply:
It seemed redundant, just saw it on stack overflow like that.  Anyways I removed it